### PR TITLE
feat(react-scheduler): highlight today DayScale cell

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-scheduler-material-ui/src/templates/views/vertical/day-scale/cell.jsx [hunk 1]
```diff
@@ -33,10 +37,20 @@ const CellBase = ({
...
+          [classes.highlightCell]: today,
+        })}
+      >
         {currentDate.format('ddd')}
       </p>
-      <span className={classes.dayOfMonth}>
+      <span
+        className={classNames({
+          [classes.dayOfMonth]: true,
+          [classes.highlightCell]: today,
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler-material-ui/src/templates/views/vertical/day-scale/cell.jsx]: What about a test for the `highlightCell` class?

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ Field | Type | Description
 ------|------|------------
 startDate | Date | Specifies the cell start time.
 endDate | Date | Specifies the cell end time.
+today | boolean | Specifies the cell's date is today.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: Is it necessary for WeekView?

File: packages/dx-react-scheduler/docs/reference/day-view.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ Field | Type | Description
 ------|------|------------
 startDate | Date | Specifies the cell start time.
 endDate | Date | Specifies the cell end time.
-
+today | boolean | Specifies the cell's date is today.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/day-view.md]: The same about 'today' here.

File: packages/dx-react-scheduler/docs/reference/week-view.md (hunk 2 above)
Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/reference/week-view.md]: This is the `DayView.CellData` interface. The `cellsData` function always returns this interface.

**Pull request merged**